sdktests/common_tests_poll_request.go

```diff
@@ -212,3 +212,164 @@ func (c CommonPollingTests) RequestContextProperties(t *ldtest.T, getPath string
 		}
 	})
 }
+
+func (c CommonPollingTests) InitialRequestIncludesCorrectEtag(t *ldtest.T) {
+	contexts := data.NewContextFactory("etag-header")
+
+	t.Run("e-tag", func(t *ldtest.T) {
+		t.Run("is not set on initial request", func(t *ldtest.T) {
+			for _, method := range c.availableFlagRequestMethods() {
+				context := contexts.NextUniqueContext()
+
+				dataSource := NewSDKDataSource(t, nil, DataSourceOptionPolling())
+				dataSource.pollingService.SetEtag(context.FullyQualifiedKey())
+
+				client := NewSDKClient(t, c.baseSDKConfigurationPlus(
+					WithClientSideInitialContext(context),
+					c.withFlagRequestMethod(method),
+					dataSource,
+				)...)
+
+				request := dataSource.Endpoint().RequireConnection(t, time.Second)
```

Comment:
is the `time.Second` already in some constant (somewhere?)

Reply:
Discussed that this should be fixed in a dedicated PR since it's inconsistent in many other places.